src/integrationTest/java/org/opentripplanner/base/ResponseStructureIT.java

```diff
@@ -61,84 +61,36 @@ private void assertPlan(TripPlan tripPlan) {
     }
 
     private void assertItinerary0(Itinerary itinerary) {
-        assertThat(itinerary.itineraryType, equalTo("WALK+MOTORBIKE+TRANSIT"));
+        assertThat(itinerary.itineraryType, equalTo("WALK+TRANSIT"));
```

Comment:
Nie podobają mi się te zmiany, choć rozumiem, że są konieczne. Tylko czemu trasa zero też się zmieniła? Pierwsza powinna być bez zmian

Reply:
Czyżby to ten moment kiedy ten test się przydaje?

Reply:
Trasa zero to jest trasa zero po posortowaniu przez planner według `duration`, a nie pierwsza znaleziona.  Prawdopodobnie tutaj objawia się to, że waga != czas przejazdu, bo nagle dostajemy szybszą trasę (która pewnie startuje później) samą komunikacją, niż komunikacją i skuterem po tym jak zabroniliśmy używania tego providera.